### PR TITLE
Configurable table renderers

### LINKED ISSUE #8704: Configurable table renderers
- improve table configuration (fine grained overrides)
- table cell component rendering

### PR DESCRIPTION
**Pull request by tobi-or-not-tobi**
closes #8704

### CODE REVIEW

File: feature-libs/my-account/organization/src/components/shared/organization-table/organization-table.module.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    AmountCellComponent,
+    DateRangeCellComponent,
+    LimitCellComponent,
+    RolesCellComponent,
+    StatusCellComponent,
+    UnitCellComponent,
+  ],
+})
+export class OrganizationCellModule {}
```
Comment (dunqan) [inline feature-libs/my-account/organization/src/components/shared/organization-table/organization-table.module.ts]: Should be `OrganizationTableModule`?

File: projects/storefrontlib/src/shared/components/table/table-renderer.service.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  Table,
+  TableDataOutletContext,
+  TableHeaderOutletContext,
+  TableOptions,
+} from './table.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TableRendererService {
```
Comment (dunqan) [inline projects/storefrontlib/src/shared/components/table/table-renderer.service.ts:23]: Would be good to add some description to the service and its role.

File: projects/storefrontlib/src/shared/components/table/table.component.ts [hunk 3]
```diff
@@ -52,13 +59,18 @@ export class TableComponent {
   @Input()
   set dataset(dataset: Table) {
     this._dataset = dataset;
+    // consider decoupling type or structure to avoid recreation of table cell components
+    // test breakpoint specific cell components
```
Comment (dunqan) [inline projects/storefrontlib/src/shared/components/table/table.component.ts]: do we want to keep those comments?

**Review by dunqan — APPROVED** [review]

**Review by Pucek9 — CHANGES_REQUESTED** [review]
- missing translation in users/details or edit [organization:user.orgUnit]
- I can't open address details (NullInjectorError: No provider for CurrentUnitAddressService!)
- in details pages, while open sub-page there is missing active class styling (grey or red background)

**Comment by tobi-or-not-tobi** [comment]
closing as this has been added by #8924 directly

**Pull request closed without merging**